src/chat.rs

```diff
@@ -3470,7 +3481,7 @@ pub async fn create_group_chat(
 
     let chat_id = ChatId::new(u32::try_from(row_id)?);
     if !is_contact_in_chat(context, chat_id, ContactId::SELF).await? {
-        add_to_chat_contacts_table(context, chat_id, &[ContactId::SELF]).await?;
+        add_to_chat_contacts_table(context, time(), chat_id, &[ContactId::SELF]).await?;
```

Comment:
Better to use `timestamp` probably, `time()` may be even less than `created_timestamp` which would look strange at least